osi_trafficcommand.proto

```diff
@@ -461,8 +480,181 @@ message SpeedAction
 
 }
 
+// \brief Longitudinal Distance Action
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LongitudinalDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+
+message LongitudinalDistanceAction 
+{
+    // Action Header of the message
+    //
+	optional ActionHeader action_header = 1;
+	
+    // Unique Id of the refence traffic participant to which the distance is defined.
+    //
+	optional Identifier target_traffic_participant_id = 2;
+	
+	// The distance to reach along the lane
+	//
+	// Unit: m
+	//
+	optional double distance = 3;
+	
+	// Determine the measurement of distance between TrafficParticipant
+	// True: Lateral distance is measured using the distance between closest bounding box points.
+    // False: Reference point distance is used
+    //
+	optional bool freespace = 4;
+	
+	// Define wheather the traffic participant should only reach the distance once 
+	// or follow.
+	// If set to true the traffic particpant shall approach the reference participant
+	// and follow woth the distance specified until the action is aborted.
+	// If set to false the action end as soon as the traffic participant has reached
+	// the 
+	optional bool follow = 5;
+	
+	// Parameter that assings either unlimited dynamics (if omitted) 
+    // or limited maxAcceleration/maxDeceleration/maxSpeed to the action.
+    //
+	optional DynamicConstraints dynamic_constraints = 6;
+}
+
+
+// \brief Lateral Distance Action 
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LateralDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+message LateralDistanceAction
+{
+    // Action Header of the message
+    //
+	optional ActionHeader action_header = 1;
+	
+    // Id of the reference entity the lateral distance shall be kept to.
+    //
+	optional Identifier target_traffic_participant_id = 2;
+	
+    // Lateral distance value. Unit: m; Range: [0...inf]
+    //
+	optional double distance = 3;
+
+	// True: Lateral distance is measured using the distance between closest bounding box points.
+    // False: Reference point distance is used
+    //
+	optional bool freespace = 4;
+
+	// If false, the action ends when the target distance is reached.
+    // If true it does not end and can only be stopped.
+    //
+	optional bool continuous = 5;
```

Comment:
I think we could harmonize the name of this bool with the LongitudinalDistanceAction and call it either continous or follow.

Reply:
Resolved in [a71b8cf](https://github.com/OpenSimulationInterface/open-simulation-interface/pull/453/commits/a71b8cf81ad6a7f0b8124ab8cd4e9d75e0766cad)

Reply:
thanks looks good to  me now.